Containerfile.api-debug

```diff
@@ -0,0 +1,34 @@
+# Builder container
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi9/go-toolset as builder
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+USER 0
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -gcflags="all=-N -l" -o /planner-api cmd/planner-api/*.go
+
+# Install Delve
+RUN GOBIN=/app go install github.com/go-delve/delve/cmd/dlv@latest
+
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi9/ubi-minimal
+
+WORKDIR /app
+
+RUN curl -Lo /app/rhcos-live.x86_64.iso https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/rhcos-live.x86_64.iso
+COPY /data /app/data/
+COPY --from=builder /planner-api /app/
+COPY --from=builder /app/dlv /app/
+
+# Use non-root user
+RUN chown -R 1001:0 /app
+USER 1001
+
+# Expose ports
+EXPOSE 3443 40000
+
+# Run the server with Delve
+ENTRYPOINT ["/bin/bash", "-c", "/app/dlv --continue --listen=:40000 --headless=true --api-version=2 --accept-multiclient exec /app/planner-api run"]
```

Comment:
Is it possible to use the same entrypoint structure like in other container files?
For example, avoid using `"/bin/bash", "-c"`?

Reply:
We can! We copy-pasted this from `Containerfile.api` where we used `bash -c` as well. Let's remove it from both.